packages/nimbus/src/components/date-range-picker/date-range-picker.dev.mdx

```diff
@@ -0,0 +1,235 @@
+        width="8000" 
+      />
+      <Text fontSize="sm">
+        Selected: {value ? `${value.start} to ${value.end}` : 'None'}
+      </Text>
+    </Stack>
+  );
+}
+```
+
+## Testing Strategies and Best Practices
+
+The DateRangePicker component requires comprehensive testing across multiple interaction patterns.
+
+#### 1. Unit Testing with React Testing Library
+
+```tsx
+import { render, screen } from '@testing-library/react';
+import { DateRangePicker } from '@commercetools/nimbus';
+
+describe('DateRangePicker', () => {
+  it('renders start and end date inputs', () => {
+    render(<DateRangePicker />);
+    const dateInputs = screen.getAllByRole('spinbutton');
+    expect(dateInputs).toHaveLength(6); // 3 segments per date input
+  });
+
+  it('shows calendar and clear buttons', () => {
+    render(<DateRangePicker />);
+    expect(screen.getByRole('button', { name: /open calendar/i })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /clear selection/i })).toBeInTheDocument();
+  });
+});
+```
+
+#### 2. Storybook Integration for Testing
+
+Leverage existing stories for comprehensive visual and interaction testing:
+
+```tsx
+export const InteractionTest: Story = {
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    
+    // Test calendar opening
+    const calendarButton = await canvas.findByRole('button', { name: /calendar/i });
+    await userEvent.click(calendarButton);
+    
+    // Verify calendar popover appears
+    await waitFor(() => {
+      const calendar = within(document.body).queryByRole('application');
+      expect(calendar).toBeInTheDocument();
+    });
+    
+    // Test date range selection in calendar
+    const dates = within(document.body).getAllByRole('gridcell');
+    await userEvent.click(dates[10]); // Start date
+    await userEvent.click(dates[15]); // End date
+    
+    // Verify selection reflected in inputs
+    const segments = canvas.getAllByRole('spinbutton');
+    expect(segments[0]).toHaveValue('10');
+  },
+};
+```
+
+#### 3. Regression tests
+
+[Placeholder for regression test strategy]
+
+Visual regression with Chromatic
+Behavior regression with interaction tests
+API contract testing
+
+### 4. Accessibility requirements
+
+- **Keyboard navigation**: Tab between inputs, arrow keys within calendar, Enter/Space for selection
+- **Screen reader support**: Announces selected dates, range information, and validation states  
+- **ARIA attributes**: Proper roles, labels, and state announcements for calendar and inputs
+- **Color contrast**: Meets WCAG AA standards for all interactive elements and text
+
+## Quick Reference Summary
+
+### Design Guidelines
+
+- **Flexible input methods**: Support both manual typing and calendar selection for user preference
+- **Clear visual feedback**: Highlight selected date ranges with appropriate visual indicators
+- **Validation integration**: Provide immediate feedback for invalid dates or ranges
```

Comment:
I'm not sure this is necessary at all anymore since they are on a separate tab on the same page